notification/fhir-notification-kafka/src/main/java/com/ibm/fhir/notifications/kafka/impl/FHIRNotificationKafkaPublisher.java

```diff
@@ -121,16 +121,16 @@ public void notify(FHIRNotificationEvent event) throws FHIRNotificationException
                 LOG.fine("Publishing kafka notification event to topic '" + topicId + "',\nmessage: " + jsonString);
             }
 
-            boolean aysnc = FHIRConfigHelper.getBooleanProperty(FHIRConfiguration.PROPERTY_KAFKA_SYNC, false);
-            if (aysnc) {
+            boolean sysnc = FHIRConfigHelper.getBooleanProperty(FHIRConfiguration.PROPERTY_KAFKA_SYNC, false);
+            if (sysnc) {
```

Comment:
Still two issues, I think.
1) Variable "sysnc" is misspelled.
2) The log message on line 128 makes it seem like you have the "sync" and "async" if branches backwards.

Reply:
Fixed.  Thanks